### PR TITLE
Add initial details on MQTT Broker service

### PR DESCRIPTION
**Pull request by hardillb**
Do not merge until Thursday

## Description

Docs for new MQTT Broker service

## Related Issue(s)

#1350 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [x] Documentation has been updated
    - [ ] Upgrade instructions
    - [x] Configuration details
    - [x] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: docs/cloud/introduction.md [hunk 1]
```diff
@@ -184,6 +185,29 @@ FlowFuse does not provide an MQTT broker for general use by Node-RED instances.
 Project Nodes can be used to easily pass messages between Node-RED instances running in the
 platform.
 
+#### Enterprise Team Broker
+
+Enterprise level teams come with their own MQTT broker. You can provision clients from the broker tab in the left hand menu.
+
+Teams will initially come with up to 20 clients, with the ability to add more coming in later releases.
```
Comment (knolleary) [inline docs/cloud/introduction.md:192]:
```suggestion
Teams can register up to 20 clients as part of their plan. The ability to purchase additional packs of clients will come in a future release.
```

**Review by knolleary — APPROVED** [review]
I think we need to add more around working with the broker from within Node-RED instances - as that will require registering a client per NR instance.

But we can iterate on that. Want to unblock the other bits we need to publish.

**Comment by hardillb** [comment]
Need to update the image here, will do that now

**Comment by knolleary** [comment]
Merge once the image is updated

**Comment by hardillb** [comment]
Changes added

**Review by knolleary — APPROVED** [review]

**Pull request merged**